views/help/forms/form_bug.hbs

```diff
@@ -11,57 +11,48 @@
+			<option value="Einstellungen">{{$t "help.chooseTopicArea.text.pSettings"}}</option>
+			<option value="Hilfebereich">{{$t "help.chooseTopicArea.text.pHelparea"}}</option>
+			<option value="IDM">{{$t "help.chooseTopicArea.text.pIdm"}}</option>
+			<option value="Kalendar">{{$t "help.chooseTopicArea.text.pCalendar"}}</option>
+			<option value="Kurse">{{$t "help.chooseTopicArea.text.pCourses"}}</option>
+			<option value="KursThemen">{{$t "help.chooseTopicArea.text.pTopics"}}</option>
 			<option value="Lern-Store">{{$t "help.chooseTopicArea.text.pLernStore"}}</option>
-			<optgroup label="{{$t "help.chooseTopicArea.text.pManagement"}}">
-				<option value="Verwaltung-Allgemein">{{$t "help.chooseTopicArea.text.pGeneral"}}</option>
-				<option value="Registrierung-Login">{{$t "help.chooseTopicArea.text.pRegisterLogin"}}</option>
-				<option value="Berechtigungen">{{$t "help.chooseTopicArea.text.pPermissions"}}</option>
-				<option value="IDM">{{$t "help.chooseTopicArea.text.pIdm"}}</option>
-			</optgroup>
-			<optgroup label="{{$t "help.chooseTopicArea.text.pExtra"}}">
-				<option value="Extra-Allgemein">{{$t "help.chooseTopicArea.text.pGeneral"}}</option>
-				<option value="Apps">{{$t "help.chooseTopicArea.text.pApps"}}</option>
-				<option value="Benachrichtigungen">{{$t "help.chooseTopicArea.text.pNotifications"}}</option>
-				<option value="Messenger">{{$t "help.chooseTopicArea.text.pMessenger"}}</option>
+			<option value="Lern-Store-Content">{{$t "help.chooseTopicArea.text.pLernStoreContent"}}</option>
+			<option value="Login">{{$t "help.chooseTopicArea.text.pLogin"}}</option>
+			<option value="MobileApps">{{$t "help.chooseTopicArea.text.pApps"}}</option>
+			<option value="Neuigkeiten">{{$t "help.chooseTopicArea.text.pNews"}}</option>
+			<optgroup label={{$t "help.chooseTopicArea.text.pTools"}}>
+				<option value="BBB-Videokonferenz">{{$t "help.chooseTopicArea.text.pBBB"}}</option>
+				<option value="Libre-Office">{{$t "help.chooseTopicArea.text.pLibreOffice"}}</option>
+				<option value="Matrix-Messenger">{{$t "help.chooseTopicArea.text.pMatrixMessenger"}}</option>
+				<option value="Rocket-Chat">{{$t "help.chooseTopicArea.text.pRocketChat"}}</option>
+				<option value="Tools-Editor">{{$t "help.chooseTopicArea.text.pEditor"}}</option>
+				<option value="Tools-Andere">{{$t "help.chooseTopicArea.text.pToolsOther"}}</option>
 			</optgroup>
+			<option value="Registrierung">{{$t "help.chooseTopicArea.text.pRegister"}}</option>
+			<option value="Berechtigungen">{{$t "help.chooseTopicArea.text.pPermissions"}}</option>
+			<option value="SHD">{{$t "help.chooseTopicArea.text.pSHD"}}</option>
+			<option value="Teams">{{$t "help.chooseTopicArea.text.pTeams"}}</option>
+			<option value="Teilen">{{$t "help.chooseTopicArea.text.pSharing"}}</option>
+			<option value="Verwaltung">{{$t "help.chooseTopicArea.text.pManagement"}}</option>
 			<option value="Sonstige">{{$t "help.chooseTopicArea.text.pOther"}}</option>
```

Comment:
was the order of the elements agreed? Without optgrup, also not alphabetically, is more difficult to find the topic you are looking for.

Reply:
Yes it was agreed on, and it is alphabetically. Optgroups don't make much sense anymore since we are using the JIRA components now where grouping doesn't make much sense in many cases.